libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/RouteHighlightOverlay.kt

```diff
@@ -23,14 +23,20 @@ import org.osmdroid.views.overlay.Overlay
 
 
 class RouteHighlightOverlay(context: Context, private val mapView: CycleMapView) : Overlay() {
+// This class highlights the current section of route and displays the turn instruction at the top,
+// either in LiveRide, or when the user is pressing the left and right arrow buttons to go through the route.
+// Alternatively, it shows the route summary at the top of the screen.
 
     private var current: Segment? = null
+    private var currentAlt: Journey? = null
 
     private val routeSummaryInfo: TextView
     private val routeNowIcon: ImageView
     private val prevButton: FloatingActionButton
     private val nextButton: FloatingActionButton
 
+    private val altText = context.getString(R.string.alt_route_details)
```

Comment:
I suggest appending a space here (rather than in the various `strings.xml` translations!) and naming it something like `private val altRoutePrefix` instead.

Reply:
The space is already in Segment.summaryText, so no need.  I'll rename the val.